<ui version="4.0" >
 <class>SelfTestDialog</class>
 <widget class="QDialog" name="SelfTestDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Kleopatra Self-Test Results</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string comment="@info" >&lt;para>These are the results of the Kleopatra self-test suite. Details are available in tooltips.&lt;/para>&lt;para>Note that all but the first failure might be consecutive.&lt;/para></string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible" >
      <bool>false</bool>
     </property>
     <widget class="QTreeView" name="resultsTV" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QGroupBox" name="proposedCorrectiveActionGB" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="title" >
       <string>Proposed Corrective Action</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="Kleo::ScrollArea" name="scrollArea" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy" >
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>336</width>
            <height>100</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="proposedCorrectiveActionLB" >
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="doItPB" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Do It!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kleo::ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>utils/scrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelfTestDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelfTestDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doItPB</sender>
   <signal>clicked()</signal>
   <receiver>SelfTestDialog</receiver>
   <slot>slotDoItClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>337</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel" >
     <x>375</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotDoItClicked()</slot>
 </slots>
</ui>
